app/views/layouts/_dark_mode_styles.html.erb

```diff
@@ -0,0 +1,3 @@
+<% if theme.dark_mode? %>
+  <%= stylesheet_link_tag 'dark-mode', media: 'all' %>
```

Comment:
It's worth mentioning we'll be getting rid of the dark mode stylesheet in a follow up PR in favour of using [Tailwinds dark variant](https://v2.tailwindcss.com/docs/dark-mode). Thats step two in [the plan](https://github.com/TheOdinProject/top-meta/issues/235) and this PR was getting big enough 😆